### PR TITLE
adds a noise cancelling sampler for Honeycomb

### PR DESCRIPTION
**Pull request by decimalator**

### CODE REVIEW

File: config/initializers/honeycomb.rb [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
-    send_data.action_controller
-    deliver.action_mailer
-  ].freeze
+# Honeycomb integration
 
-  config.client = Libhoney::NullClient.new if Rails.env.development? || ENV.key?("HONEYCOMB_DEBUG")
+if Rails.env.development? || ENV.key?("HONEYCOMB_DEBUG")
+  Honeycomb.configure do |config|
+    config.client = Libhoney::NullClient.new
+    #config.client = Libhoney::TestClient.new
```
Comment (CGillen) [inline config/initializers/honeycomb.rb:6]: Could you drop this commented line?

File: config/initializers/honeycomb.rb [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+      render_collection.action_view
+      process_action.action_controller
+      send_file.action_controller
+      send_data.action_controller
+      deliver.action_mailer
+    ].freeze
+
+    # Scrub unused data to save space in Honeycomb
+    config.presend_hook do |fields|
+      #fields["global.build_id"] = ApplicationConfig["HEROKU_SLUG_COMMIT"]
```
Comment (CGillen) [inline config/initializers/honeycomb.rb:26]: And this one

File: config/initializers/honeycomb.rb (hunk 1 above)
Comment (decimalator) [reply config/initializers/honeycomb.rb]: done

File: config/initializers/honeycomb.rb (hunk 2 above)
Comment (decimalator) [reply config/initializers/honeycomb.rb]: done

**Pull request merged**